docker-images/bundle/Makefile

```diff
@@ -0,0 +1,70 @@
+PROJECT_NAME=bundle
```

Comment:
Maybe we can call it `olm-bundle` (and rename the directory as well)? `bundle` is very generic term (for me the YAML installation files are bundle)

Reply:
I agree, there are so many bundles in the world